.travis.yml

```diff
@@ -1,38 +1,47 @@
-dist: bionic
-language: ruby
-rvm:
-  - 2.5.3
-cache:
-  - bundler
-addons:
-  postgresql: 9.5
-  apt:
-    packages:
-      - firefox-geckodriver
-      - libarchive-dev
-      - libgd-dev
-      - libffi-dev
-      - libbz2-dev
-services:
-  - memcached
-before_script:
-  - sed -e 's/ IMMUTABLE / /' -e "/^--/d" db/structure.sql > db/structure.expected
-  - psql -U postgres -c "CREATE DATABASE openstreetmap"
-  - psql -U postgres -c "CREATE EXTENSION btree_gist" openstreetmap
-  - psql -U postgres -f db/functions/functions.sql openstreetmap
-  - cp config/travis.database.yml config/database.yml
-  - cp config/example.storage.yml config/storage.yml
-  - touch config/settings.local.yml
-  - echo -e "---\nmemcache_servers:\n  - 127.0.0.1" > config/settings/test.local.yml
-  - bundle exec rake db:migrate
-  - bundle exec rake i18n:js:export
-  - bundle exec rake yarn:install
-script:
-  - bundle exec rubocop -f fuubar
-  - bundle exec rake eslint
-  - bundle exec erblint .
-  - bundle exec brakeman -q
-  - bundle exec rake db:structure:dump
-  - sed -e "/idle_in_transaction_session_timeout/d" -e 's/ IMMUTABLE / /' -e "/^--/d" db/structure.sql > db/structure.actual
-  - diff -uw db/structure.expected db/structure.actual
-  - bundle exec rake test:db
+jobs:
+  include:
+  - name: "Locale Authorship"
+    install: skip
```

Comment:
This one executes on "xenial". For consistency, maybe add "dist: bionic" as well?